episode2.py

```diff
@@ -112,18 +113,34 @@ def update(self, dt):
             item.velocity += g * dt
 
         if len(self.drugs) < 15 and random.randint(0, 100) < 10:
-            drug = arcade.Sprite("img/coin.png", SPRITE_SCALING_COIN)
-            drug.score = 1
-            drug.velocity = random.randint(10, 25) * g
-            drug.center_x = random.randint(0, self.width)
-            drug.center_y = random.randint(self.height - 100, self.height)
-            self.drugs.append(drug)
+            if self.time_left > 20:
```

Comment:
DRY. This can be done in one function with different arguments (image and time) 
+ add constants instead of hardcoded numbers